src/python/grapl-http-service/requirements.txt

```diff
@@ -0,0 +1,6 @@
+pytest
+pytest-cov
+hypothesis
+pyjwt
+chalice
+boto3_stubs
```

Comment:
Once the docker build is reworked to use Pants, this file won't be needed; it's handled by https://github.com/graplsecurity/grapl/blob/main/3rdparty/python/requirements.txt